### PR TITLE
Working code for 2016 show

### PR DESCRIPTION
**Pull request by bookdude13**
I know this got big. Did not do a good job syncing with master while doing this work. If someone could give this a once-over, that would be fantastic, since the odds of me doing something strange or inconsistent are fairly high. Keep in mind that this code did indeed run correctly for the show, so the logic should be sound for expected use cases. Edge cases that I missed are going to be addressed as tests are written (although bringing them up here would be great so I know which tests to focus on more).

### CODE REVIEW

File: layouts/key.pub [hunk 1]
```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
```
Comment (computergeek125) [inline layouts/key.pub]: Sure you want public keys in a repo?

File: src/sequence.rs [hunk 2]
```diff
@@ -52,20 +49,19 @@ pub fn new_vixen_sequence<P: AsRef<Path>, CD: ChannelDao, DD: DataDao, FD: Fixtu
     let music_duration_sec = try!(get_music_duration_sec(&music_file_path));
     
     // Try to copy music file into music directory
-    try!(copy_music_file(&music_file_path, &music_file_name, "Music"));
+    //try!(copy_music_file(&music_file_path, &music_file_name, "Music"));
```
Comment (computergeek125) [inline src/sequence.rs]: no copy?

**Review by computergeek125 — COMMENTED** [review]
Looks good, except for one potential issue and a possible watchpoint.  If you justify it or resolve it I'll approve.

File: src/sequence.rs (hunk 2 above)
Comment (bookdude13) [reply src/sequence.rs]: The copy was an idea I had that didn't really help, and it got annoying, so I commented it out. Will properly remove

File: layouts/key.pub (hunk 1 above)
Comment (bookdude13) [reply layouts/key.pub]: Actually, I don't. Whoops.

**Comment by bookdude13** [comment]
Changes made

**Review by computergeek125 — APPROVED** [review]
